set.mm

```diff
@@ -21440,9 +21444,10 @@ proposition with a distinct variable (closed form of ~ nfsb4 ).
 
   ${
     $d y w z $.  $d x w $.
-    $( Substitution applied to an atomic wff.  (Contributed by Raph Levien and
-       FL, 4-Dec-2005.)  Remove dependency on ~ ax-11 .  (Revised by Wolf
-       Lammen, 21-Sep-2018.) $)
+    $( Substitution applied to an atomic wff.  For a version requiring disjoint
```

Comment:
to an atomic wff -> to an equality
(an atomic formula can also be of the form `x e. y`)
It can even be "Substitution in an equality."